superduperdb/backends/compute/dask.py

```diff
@@ -23,14 +24,12 @@ def __init__(
         serializers: t.Optional[t.Sequence[t.Callable]] = None,
         deserializers: t.Optional[t.Sequence[t.Callable]] = None,
         local: bool = False,
-        envs: t.Optional[t.Dict[str, t.Any]] = None,
         **kwargs,
     ):
-        envs = envs or {}
         self.futures_collection: t.Dict[str, distributed.Future] = {}
 
         if local:
-            cluster = distributed.LocalCluster(env=envs)
+            cluster = distributed.LocalCluster(processes=False)
```

Comment:
rather than processes=False, 
better would be to take **kwargs from user, since user might want a multiprocesses setup for local dask